lib/autoflow.coffee

```diff
@@ -68,11 +68,22 @@ module.exports =
       currentLine = []
       currentLineLength = linePrefixTabExpanded.length
 
+      notFirstLine = false
```

Comment:
I think it would be easier to understand if this was switched: `firstLine = true`